tutorials/dlp-hybrid-inspect/index.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Cloud Data Loss Prevention (DLP) Hybrid Inspect demonstration for SQL databases using JDBC
+description: Demonstrates how to inspect SQL table data using Cloud Data Loss Prevention with Hybrid Inspect.
+author: scellis,crwilson
+tags: Cloud DLP, Java, PII
+date_published: 2020-08-21
+---
+
```

Comment:
Before jumping into the configurationa and build section, give an introduction that tells who this document is for and what the reader will learn and accomplish.

Reply:
Updated the wording here. Thoughts?